testdata/cloud/misc/vsphere-creds_test.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+data:
+  vcsa2-qe.vmware.devcluster.openshift.com.password: CWdycjRiQFhqpjA2
+  vcsa2-qe.vmware.devcluster.openshift.com.username: QSRtbW4pc1RyYXRvckBlMmUdvG9jZWw=
```

Comment:
Are they random passwords?(just in case)

Reply:
yes ..